it/mail/chrome/messenger/safeMode.dtd

```diff
@@ -43,26 +43,26 @@
 <!ENTITY safeModeOptionsWindow.width       "width: 49em;">
 <!ENTITY safeModeOptionsDialog.title       "&brandShortName; Trouble Shooting">
 
-<!ENTITY safeModeOptionsDescription2.label "If you are experiencing difficulty with Postbox, try restarting 
+<!ENTITY safeModeOptionsDescription2.label "If you are experiencing difficulty with Postbox, try restarting
 with one or more of these trouble shooting options selected:">
 
 
-<!ENTITY disableAddons.label              "Permamently Disable all Add-ons">
```

Comment:
@albertovlcbo We need line 46 translated, suggestion is:
Se stai avendo difficoltà con Postbox, prova a riavviare con una o più delle opzioni per la risoluzione dei problemi selezionate: